lib/msal-node/src/client/ClientApplication.ts

```diff
@@ -53,6 +53,7 @@ import { CommonUsernamePasswordRequest } from "../request/CommonUsernamePassword
 import { NodeAuthError } from "../error/NodeAuthError.js";
 import { UsernamePasswordClient } from "./UsernamePasswordClient.js";
 import { getAuthCodeRequestUrl } from "../protocol/Authorize.js";
+import { StubPerformanceClient } from "@azure/msal-common";
```

Comment:
Common imports need to use subpaths
```suggestion
import { StubPerformanceClient } from "@azure/msal-common/node";
```